package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "md-form-builder",
-  "version": "0.13.3",
+  "version": "0.13.4",
```

Comment:
This should be a major change but since it's still a 0.xx we should bump the minor number according to semver.